crates/nu-command/src/filters/get.rs

```diff
@@ -18,6 +18,13 @@ impl Command for Get {
         "Extract data using a cell path."
     }
 
+    fn extra_usage(&self) -> &str {
+        r#"This is equivalent to using the cell path access syntax: `$env.OS` is the same as `$env | get OS`.
+
+If multiple column names, row numbers, or cell paths are given, this will behave
+very similarly to `select`."#
```

Comment:
I don't think this is accurate. It does not behave similarly to select. `get` pulls things out of tables. So, if you were to do `ls | get name type` you get a list of two lists. A list of names and a list of types. If you do a `ls | select name type` you'll maintain the table, aka list of records, but it will only have name and type.

So, I think the wording on this last sentence is a little misleading and misses the fundamental difference between `get` and `select`.

Reply:
Good point - it's true for row numbers but not column names.